js/interface.js

```diff
@@ -2257,6 +2276,21 @@ $('#fl-store-2fa-select, #fl-ent-2fa-select').on('change', function (e) {
   socket.to(socketClientId).emit('aab.apple.login.2fa.device', e.target.value);
 });
 
+Fliplet().then(function () {
+  checkDemoUser();
+});
+
+// This listener need that we can understand that it is a user entering data but not an any password managers.
+$('#fl-store-revDemoUser').on('keyup', function (event) {
+  userInput = event && event.key || false;
+});
+
+// After user blur from 'demo user' field we check again to make sure that the field is empty. 
+// If field is empty we remove required attribute.
+$('#fl-store-revDemoUser').on('change', function () {
```

Comment:
Here are all the events related to the form field that I can think of:

1. User typing
1. User pasting through Ctrl/Cmd + V
1. User pasting through contextual/browser menu
1. User filling in data through browser suggestion dropdown
1. User auto-filling through password manager
1. Password manager auto-fill
1. Browser auto-fill

You should be able to use the `input` event to capture all the changes above.

```suggestion
$('#fl-store-revDemoUser').on('input', function () {
```

Reply:
@tonytlwu 
1. As I understand password manager auto-fill and browser auto-fill shouldn't paste any information to the text field by themselves, is that right?
2. And the user should have a chance to use the auto-fill option like on the screenshot below?
![image](https://user-images.githubusercontent.com/53430352/63248045-2d60fc80-c26f-11e9-94d7-19a173952e83.png)

Reply:
1. Yes
2. Yes

Is all of this more complicated because we're trying to avoid 1? If so, perhaps adding the following would help.

* Load submission data from API
* Load form from the submission data
* If demo user and password are not provided, set them as empty string -- I imagine that's what's missing because it's quite likely we're filling in information purely based on what **is** in the API submission data

Reply:
@tonytlwu 
We can suggest another solution.

What if we don't show this block in case if there is no data came from API?

![image](https://user-images.githubusercontent.com/53430352/63267844-965e6980-c29b-11e9-8746-4c971f2ebfca.png)

Instead, we would show a button with a label 'Fill demo user data' and when the user will click on it we show him fields above.

In this case, the user would be intended to fill this data because of that we wouldn't need to check auto-fill, the user will check data by himself when he filled them.

Will this suggestion suit the current UI flow of this component or better stick to the current solution?

Reply:
@YaroslavOvdii I like what you're suggesting, especially if that's how we designed it at the beginning, but I'd avoid changing the UI at this stage. Also, when you load an old submission, you'll need to tick the checkbox as well based on user & password data -- unnecessarily complicated.

Based on what I see, you should be able to just add another `if` block in the `loadAppStoreData()` function and attach the current validation to the `input` event. I think that should meet all the criteria?

Reply:
@tonytlwu 
Add new commit to solve this issue.